dump_env/cli.py

```diff
@@ -6,6 +6,43 @@
 from dump_env.exceptions import StrictEnvError
 
 
+def needs_quotes(value: str) -> bool:
+    """
+    Check if the value needs to be quoted.
+
+    Args:
+        value (str): The value to check.
+
+    Returns:
+        bool: True if the value needs to be quoted, False otherwise.
+    """
+    return (
+        not value
+        or ' ' in value
+        or '\n' in value
+        or '=' in value
+        or '"' in value
+        or "'" in value
```

Comment:
I am not actually 100% sure this one is needed, as `dotenv` itself says for multiline support a `"` is needed anyway, see https://github.com/theskumar/python-dotenv?tab=readme-ov-file#multiline-values